libraries/Microsoft.Bot.Builder.Azure.Blobs/BlobsTranscriptStore.cs

```diff
@@ -406,7 +421,23 @@ private async Task LogActivityToBlobClientAsync(IActivity activity, BlobClient b
                 ["RecipientId"] = activity.Recipient?.Id,
                 ["Timestamp"] = activity.Timestamp.Value.ToString("O", CultureInfo.InvariantCulture)
             };
-            await blobClient.SetMetadataAsync(metaData).ConfigureAwait(false);
```

Comment:
Note: Upgrade of `Azure.Storage.Blobs` allowed metadata to be passed in with the `UploadAsync()` call. Woohoo! One less transaction!